app/src/main/res/layout/fragment_connect.xml

```diff
@@ -18,22 +18,37 @@
     For the additional permissions granted for Salt Edge Authenticator
     under Section 7 of the GNU General Public License see THIRD_PARTY_NOTICES.md
 -->
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent">
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <com.saltedge.authenticator.sdk.web.SEWebView
-        android:id="@+id/connectWebView"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+    <data class="ConnectProviderBinding">
+
+        <import type="android.view.View" />
 
-    <include
-        android:id="@+id/fragmentConnectProcessingLayout"
-        layout="@layout/fragment_connect_processing" />
+        <variable
+            name="viewModel"
+            type="com.saltedge.authenticator.features.connections.create.ConnectProviderViewModel" />
+    </data>
 
-    <com.saltedge.authenticator.widget.CompleteView
-        android:id="@+id/completeView"
+    <FrameLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@android:color/white" />
-</FrameLayout>
+        android:layout_height="match_parent">
+
+        <com.saltedge.authenticator.sdk.web.SEWebView
+            android:id="@+id/connectWebView"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:visibility="@{viewModel.shouldShowWebViewVisibility}" />
+
+        <include
```

Comment:
fragment_connect_processing is equal to complete_view. why we need it?

Reply:
The design is still being updated, so I think we can leave it like this for now, so as not to delay this `pr`